### DISCUSSION TITLE
Links to top-level pages not working?

### DISCUSSION THREAD

**Discussion by mikro2nd** [zim-desktop-wiki/zim-desktop-wiki#2467] [General]
Since the last upgrade to 0.75.2, "``links to a page in the top level of the notebook``" have stopped working for me. i.e. the `[[:TopLevelPage]]` syntax is not being recognised and does not automatically create a link to the page as it gets typed as a simple `:TopLevelPage` text.

On the other hand, linking to an *already existing* top-level page (I have CamelCaseLinking enabled, of course ;)) now links *without* using the colon syntax -- an improvement I've long wished for! I mention this because I suspect that the two behaviours are related.

Am I the only one seeing this misbehaviour? (Don't wish to log a bug unless it really is one. If it's merely an env issue I'm suffering from then I will look elsewhere than Zim for a fix.)

**Comment by introt** [comment]
`:TopLevelPage` no longer autoformats as a link, `[[:TopLevelPage]]` works after hitting Ctrl+R to reload. If a sibling named TopLevelPage exists, the Link dialog produces `[[:TopLevelPage]]`. Imo `:TopLevelPage` should still autoformat, feel free to file a ticket.

**Comment by mikro2nd** [comment]
Bug logged: https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/2469
